controls/radspreadsheet/features/comments.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Threaded Comments
+page_title: Threaded Comments 
+slug: radspreadsheet-features-comments
+tags: thread, comments, xlsx, 
+published: True
+position: 13
+---
+
+# Comments
+
+Since R1 2022 the Spreadsheet control supports working with comments.
+
+Comments are used for leaving information about a cell's data, where each comment can be replied to, thus creating a thread. All Comments can be found in the __Comments__ collection of the worksheet. More information about using Comments in code is available here: [SpreadProcessing Comments](https://docs.telerik.com/devtools/document-processing/libraries/radspreadprocessing/features/comments). 
+
+#### Figure 1: Sample Comment
+
+![RadSpreadProcessing Comments](images/RadSpreadProcessing_Comments_01.png)
+
+### Using Comments in your Application
+
+RadSpreadsheet enables working with Comments through its UI. There are two ways for adding Comments, via the context menu and from the Comments group on the Review tab.
+
+#### Figure 2: Add Comment from the context menu
+
+![RadSpreadProcessing Comments](images/RadSpreadProcessing_Comments_02.png)
+
+#### Figure 3: Add Comment from the Comments group
+
+![RadSpreadProcessing Comments](images/RadSpreadProcessing_Comments_03.png)
+
+As you can see in the above image, the Comments group allows you to perform the following actions:
+
+* __New Comment:__ Adds new comment to the selected cell.
+* __Reply Comment:__ Adds a reply to an existing comment to the selected cell. (only available when selected cell already contains a comment)
```

Comment:
* __Reply Comment:__ Adds a reply to an existing comment. This option is only available when the selected cell already contains a comment.